curvefs/src/client/fuse_client.cpp

```diff
@@ -186,6 +186,14 @@ CURVEFS_ERROR FuseClient::Init(const FuseClientOption &option) {
         warmupManager_->SetFsInfo(fsInfo_);
     }
 
+    FsUsedUpdater::GetInstance().Init(fsInfo_->fsid(), metaClient_);
+    StartUpdateFsUsedTask(&FsUsedUpdater::GetInstance(), 5);
```

Comment:
hardcode to 5 seconds? should be changed to a configration item?

Reply:
yes, todo later